apps/console/src/features/private-key-jwt/pages/private-key-jwt-config-edit.tsx

```diff
@@ -0,0 +1,256 @@
+                    ":id",
+                    ServerConfigurationsConstants.LOGIN_ATTEMPT_SECURITY_CONNECTOR_CATEGORY_ID
+                )
+        );
+    };
+
+    const resolveConnectorUpdateErrorMessage = (error: any): string => {
+        
+        return (
+            t("console:manage.features.jwtPrivateKeyConfiguration.notifications.error.description",
+                { description: error.response.data.description })
+        );
+    };
+    const resolveConnectorUpdateSuccessMessage = (): string => {
+
+        return (
+            t("console:manage.features.jwtPrivateKeyConfiguration.notifications.success.description")
+        );
+    };
+
+    const handleUpdateSuccess = () => {
+        dispatch(
+            addAlert({
+                description: resolveConnectorUpdateSuccessMessage(),
+                level: AlertLevels.SUCCESS,
+                message: t(
+                    "console:manage.features.jwtPrivateKeyConfiguration.notifications." +
+                    "success.message"
+                )
+            })
+        );
+    };
+
+    const handleUpdateError = (error: any) => {
+        if (error.response && error.response.data && error.response.data.detail) {
+            dispatch(
+                addAlert({
+                    description: resolveConnectorUpdateErrorMessage(error),
+                    level: AlertLevels.ERROR,
+                    message: t(
+                        "console:manage.features.jwtPrivateKeyConfiguration.notifications.error.message"
+                    )
+                })
+            );
+        } else {
+            // Generic error message
+            dispatch(
+                addAlert({
+                    description: t(
+                        "console:manage.features.jwtPrivateKeyConfiguration.notifications." +
+                        "genericError.description"
+                    ),
+                    level: AlertLevels.ERROR,
+                    message: t(
+                        "console:manage.features.jwtPrivateKeyConfiguration.notifications." +
+                        ".genericError.message"
+                    )
+                })
+            );
+        }
+    };
+
+    /**
+     * Handles token reuse toggle button
+     */
+    const handleToggle = (e: SyntheticEvent, data: CheckboxProps) => {
+        setEnableTokenReuse(data.checked);
+        const updateData: any = {
+            operation: "ADD",
+            path: "/enableTokenReuse",
+            value: data.checked
+        };
+
+        updateJWTConfig(updateData).then(() => {
+            handleUpdateSuccess();
+        }).catch((error: any) => {
+            handleUpdateError(error);
+        });
+    };
+
+    /**
+     * This renders the enable toggle.
+     */
+    const connectorToggle = (): ReactElement => {
+        return (
+            <>
+                <Checkbox
+                    label={ enableTokenReuse ? "Token Reuse Enabled" : "Token Reuse Disabled" }
```

Comment:
Shall we add i18n strings

Reply:
fixed with https://github.com/wso2/identity-apps/pull/3786/commits/dd8a9366531a26a1a44411371fe1db17b81925ae